packages/camera/example/pubspec.yaml

```diff
@@ -13,6 +13,9 @@ dependencies:
 dev_dependencies:
   flutter_test:
     sdk: flutter
+  flutter_driver:
+    sdk: flutter
+  test: 1.6.3
```

Comment:
Is this dependency actually used? I believe you only use `flutter_test`?

Reply:
It was extra, removed.